scripts/au/gnaf.sh

```diff
@@ -10,17 +10,21 @@ mkdir $TMP
 mkdir $TMP/gnaf $TMP/gnaf-admin $TMP/tablespace
 chown postgres:postgres $TMP/tablespace
 
-/etc/init.d/postgresql start
-sudo -u postgres psql -c "CREATE USER gnafun WITH CREATEUSER PASSWORD 'gnafpw'"
+/etc/init.d/postgresql start 
+sudo -u postgres psql -c "CREATE USER gnafun WITH SUPERUSER PASSWORD 'gnafpw'"
 sudo -u postgres psql -c "CREATE TABLESPACE gnafts OWNER gnafun LOCATION '$TMP/tablespace'"
 sudo -u postgres psql -c 'CREATE DATABASE gnafdb OWNER gnafun TABLESPACE gnafts'
 sudo -u postgres psql -c 'CREATE EXTENSION postgis'
 
 # fetch data/resources, cached from:
-# http://data.gov.au/dataset/bdcf5b09-89bc-47ec-9281-6b8e9ee147aa/resource/53c24b8e-4f55-4eed-a189-2fc0dcca6381/download/feb17adminboundsesrishapefileordbffile.zip
-# http://data.gov.au/dataset/19432f89-dc3a-4ef3-b943-5326ef1dbecc/resource/99b44dff-4e84-4cb7-9cbf-a68d3ebf964a/download/feb17-gnaf-pipeseperatedvalue.zip
-curl -s 'http://s3.amazonaws.com/data.openaddresses.io/cache/au/gnaf-admin-feb2017.zip' -o $TMP/gnaf-admin.zip &
-curl -s 'http://s3.amazonaws.com/data.openaddresses.io/cache/au/gnaf-feb2017.zip' -o $TMP/gnaf.zip &
+
+## https://data.gov.au/dataset/psma-administrative-boundaries
+### http://data.gov.au/dataset/bdcf5b09-89bc-47ec-9281-6b8e9ee147aa/resource/53c24b8e-4f55-4eed-a189-2fc0dcca6381/download/aug17adminboundsesrishapefileordbffile20170821151234.zip
+
+## https://data.gov.au/dataset/geocoded-national-address-file-g-naf
+### http://data.gov.au/dataset/19432f89-dc3a-4ef3-b943-5326ef1dbecc/resource/99b44dff-4e84-4cb7-9cbf-a68d3ebf964a/download/aug17gnafpipeseparatedvalue20170821153434.zip
+curl -s 'https://www.alantgeo.com.au/data.openaddresses.io/cache/au/gnaf-admin-aug2017.zip' -o $TMP/gnaf-admin.zip &
+curl -s 'https://www.alantgeo.com.au/data.openaddresses.io/cache/au/gnaf-aug2017.zip' -o $TMP/gnaf.zip &
```

Comment:
I do wonder if we need the s3 catch or can simply rely on the upstream data hosting? either way these resources need to by copied to the OA s3 cache.